src/version.h

```diff
@@ -16,19 +16,15 @@
 #define NINJA_VERSION_H_
 
 #include <string>
-using namespace std;
```

Comment:
You could rebase on https://github.com/ninja-build/ninja/pull/1614  to have this done for you.